Open_Science_New.html

```diff
@@ -69,93 +65,136 @@ <h4 style="padding-bottom:0px; margin-bottom:0px; text-shadow:-1px -1px 0 #000,
 	    <h2><font color="black">What is Open Science?</font></h2>
 	  </section>
-	  
+
 	  <!-- Big Data -->
 	  <section>
-	    <img src="images/Big_data.png">
-	  </section>
-	  
-	  <section>
-	    <img src="images/Whale.png">
+		    <a href = "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3983169/"><img src="images/Big_data.png" height="60%" width="60%" align="center"></a>
+		    <p><small>As this graph shows, the amount of data collected from <br> neuroscientific studies will only continue to increase exponentially.</small></p
+          </section>
 	  </section>
 
 	  <!-- Multi-resolution/trajectories -->
-	  <section>
-	    <img src="images/Neuroinformatics_scope.jpg" width="77%">
+         <section data-background="images/Neuroinformatics_scope.jpg" data-background-size="50%">
+	    <h2><font color="FF5733">Data Science is COMPLEX!</font></h2>
 	  </section>
-	  
-	</section>
-	
-	<section data-transition="convex" id="openscience-destination">
-	  <section>	
+
+        <!-- Benefits of Open Science -->
+        <section>
+	    <h2>Benefits of Open Science</h2>
+	    <p>Increased exposure</p>
+	    <p>Greater collaborations</p>
+	    <p>More citations</p>
+	    <p>Less money spent of patents</p>
+	    <p>New funding opportunities</p>
+	    <p>Improved reproducibility</p>
+	    <p>Enable scientific discovery</p>
+            <p><strong><font color="yellow">...It's the future!</font></strong></p>
+	  </section>
+
+        <section>
+            <h2>Aled Edwards, CEO</h2>
+            <h4>Structural Genomics Consortium</h4>
+            <p>TED Talk: The Need for Biomedical Philanthropy</p>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/Wtp7R97jz5k" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </section>
+
+        <!-- Data Publishing Not Sharing -->
+        <section data-transition="fade" data-background="images/data_publishing.png">
+	      <h1 style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;"><font color="indigo">Data Publishing<br>
+		- Not Data Sharing -</font></h1>
+            <p style="text-shadow:-1px -1px 0 #000,1px -1px 0 #000,-1px 1px 0 #000,1px 1px 0 #000;">This is an important differentiation. Data publishing calls for extensive reviewing and quality control processes.</p>
+	  </section>
+
+	  <!-- EU going Open Access -->
+        <section>
+	      <table class="reveal">
+		<tr>
+		  <td width="50%" style="vertical-align: middle">
+		    <a href="https://www.theguardian.com/science/2016/may/28/eu-ministers-2020-target-free-access-scientific-papers" target="_blank"><img src="images/Open_access.png"></a>
+		  </td>
+
+		  <td width="50%" style="vertical-align: middle;">
+		    <a href="https://www.scienceeurope.org/" target="_blank"><img src="images/ScienceEurope.png" style="border:0; border-radius:30px;"></a>
+		    <p><small><br>This intiative was officially launched as<br>
```

Comment:
@christinerogers Found this info through my research - is it accurate and relevant?

Reply:
https://htmlpreview.github.io/?https://github.com/knoll-alex/samirdas.github.io/blob/ak-Open_Science/Open_Science_New.html#/1/6